### PR TITLE
Map config cleanup

### PR DESCRIPTION
**Pull request by Cyberboss**
By removing redundant statements

### CODE REVIEW

File: _maps/boxstation.json [hunk 1]
```diff
@@ -1,8 +1,5 @@
 {
     "map_name": "Box Station",
     "map_path": "map_files/BoxStation",
-    "map_file": "BoxStation.dmm",
-    "minetype": "lavaland",
```
Comment (praisenarsie) [inline _maps/boxstation.json]: aRe YoU aSsUmInG aLl CoDeBaSeS wIlL uSe LaVaLaNd?

Comment (naltronix) [reply _maps/boxstation.json]: if not they can change the code

**Comment by Cyberboss** [comment]
mine_type literally does nothing except throw a warning because we
currently only support lavaland

**Comment by SpaceManiac** [comment]
Broken, the servers are stuck and unable to cycle maps. Because of this, I can't check, but reading the diff it kind of looks like `allow_custom_shuttles` will always be set to false (since none of the JSONs specify it).

```irc
<SpaceManiac> just click "Change Map" and see if any maps actually show up
<Kevinz000-Cloud> kk
<Kevinz000-Cloud> SpaceManiac: nope
<Kevinz000-Cloud> only the current map
```

```
Loading config file maps.txt...
transition_config is not a list!
map_config failed to validate for above reason: _maps/boxstation.json
Failed to load map config for boxstation!
transition_config is not a list!
map_config failed to validate for above reason: _maps/metastation.json
Failed to load map config for metastation!
transition_config is not a list!
map_config failed to validate for above reason: _maps/pubbystation.json
Failed to load map config for pubbystation!
transition_config is not a list!
map_config failed to validate for above reason: _maps/omegastation.json
Failed to load map config for omegastation!
transition_config is not a list!
map_config failed to validate for above reason: _maps/deltastation.json
Failed to load map config for deltastation!
transition_config is not a list!
map_config failed to validate for above reason: _maps/runtimestation.json
Failed to load map config for runtimestation!
```

**Comment by Bawhoppen** [comment]
You broke the rules

**Pull request merged**